test/spec/modules/tappxBidAdapter_spec.js

```diff
@@ -1,5 +1,6 @@
 import { assert } from 'chai';
 import { spec } from 'modules/tappxBidAdapter';
+import { getFloor } from 'modules/priceFloors';
```

Comment:
please remove this, unneeded import, you are mocking this module.

Reply:
Done! : )